src/azure-cli/azure/cli/command_modules/botservice/tests/latest/recordings/test_botservice_update_should_update_bot_properties.yaml

```diff
@@ -38,92 +38,13 @@ interactions:
       x-content-type-options:
       - nosniff
       x-ms-ratelimit-remaining-subscription-deletes:
-      - '14998'
+      - '14999'
     status:
       code: 204
       message: No Content
-- request:
```

Comment:
Not sure why this HTTP call vanished... As a precaution I've updated `azdev` and reinstalled all dependencies, but rerecording the test locally does not show this call being added back...

Link to the test in commit `7c6f19c`: https://github.com/Azure/azure-cli/pull/10584/files#diff-70958f4d3b016497951a21558049e91dR792

Reply:
@qwordy or @jiasli could someone take a look please? Thanks 😄

Reply:
Can you pass the test cases locally? What's the error message?

Reply:
Sorry for the delay @qwordy, the tests pass locally when running in recorded mode in with the `--series` flag.

Reply:
@qwordy / @yugangw-msft could I get some assistance or troubleshooting steps for this? 😄

Reply:
Sorry to reply late. I have not encountered this problem. But we can have a phone call, and maybe share your screen to show more information. I am available from 9:30 AM, UTC+8.

Reply:
I have recently got another machine that I'll pull my branch and test it on, since these generated recordings right now only work on my initial dev machine. I should have an update 10/16 afternoon UTC-7.